src/index.ts

```diff
@@ -61,50 +68,98 @@ function getExtraOutputPaths(
     s => path.relative(paths.root, path.join(paths.sources, s)),
   );
 
-  if (options?.initializablePath) {
-    outputPaths.initializable = options?.initializablePath;
+  if (options.initializablePath) {
+    outputPaths.initializable = options.initializablePath;
   }
 
   return outputPaths;
 }
 
+function getExcludeAndImportPathsForPeer(
+  solcOutput: SolcOutput,
+  peerProject: string,
+): [Set<string>, NodeTransformData[]] {
+  const data: NodeTransformData[] = [];
+  const exclude: Set<string> = new Set();
+
+  for (const [source, { ast }] of Object.entries(solcOutput.sources)) {
+    let shouldExclude = true;
+    for (const node of findAll('ContractDefinition', ast)) {
+      if (node.contractKind === 'contract') {
```

Comment:
> Do we need it for OZ Contracts?

Not really. Maybe that could replace some of the "initializable" specific logic, but its not something we absolutelly need.

Reply:
`@custom:force-transpile` is not what we need, I can't imagine a use case for it. I think the only contract where `need-not-transpile` could be relevant is `UUPSUpgradeable`. The current logic transpiles `UUPSUpgradeable`, though for this contract in particular we have a trick which is that we don't add the `*Upgradeable` suffix because the identifier already has it. Since it doesn't have any storage variables it won't get a namespace either. It does have initializers added.

Perhaps it's best to keep transpiling this contract and others simply for consistency, the initializers might be enough of a reason.

Reply:
> Also, what if they are used anywhere, and we don't have a peerImport path for them? How should we deal with that? Throw an error?

Yeah that's a good question.

If we don't need this for OZ Contracts, let's not do it.